how-to/troubleshoot/diagnose-map.md

```diff
@@ -0,0 +1,566 @@
+---
+title: TiDB Troubleshooting Map
+summary: Learns how to troubleshoot common issues in TiDB.
+category: how-to
+---
+
+# TiDB Troubleshooting Map
+
+## 1. Service Unavailable
+
+### 1.1 The client reports `Region is Unavailable` error
+
+- 1.1.1 `Region is Unavailable` is usually because a Region is not available for a period of time. You might encounter `TiKV server is busy`, or the request to TiKV fails due to `not leader` or `epoch not match`，or the request to TiKV time out. In such cases, TiDB performs a `backoff` retry mechanism. When the `backoff` exceeds a threshold (20s by default), the error will be sent to the client. Within the `backoff` threshold, this errors is not visible to the client.
```

Comment:
```suggestion
- 1.1.1 `Region is Unavailable` is usually because a Region is not available for a period of time. You might encounter `TiKV server is busy`, or the request to TiKV fails due to `not leader` or `epoch not match`, or the request to TiKV time out. In such cases, TiDB performs a `backoff` retry mechanism. When the `backoff` exceeds a threshold (20s by default), the error will be sent to the client. Within the `backoff` threshold, this errors is not visible to the client.
```